### ISSUE TITLE
GoogleAuthUiProvider error: No credentials available

### ISSUE THREAD

**Issue by eye-dee** [mirzemehdi/KMPAuth#20]
Hello

Thank you for setting up the example. I have a quick question regarding an issue I am experiencing

When clicking on sign in with google, I am getting `GoogleAuthUiProvider error: No credentials available` (let me know if you need the whole stacktrace)

I don't fully understand, if I need to do anything before being able to sign in with google

**Comment by mirzemehdi** [comment]
@eye-dee usually this happens when user doesn't have already signed in user, when testing in emulators.

This error is also same as this https://github.com/mirzemehdi/KMPAuth/issues/19. New Credential Manager doesn't support some devices, I am planning to implement old credential manager when this issue happened thanks to @stevdza-san 's suggestion.

**Comment by mirzemehdi** [comment]
Can you please let me know if this happens when you sign-in with google in device and trying again?

**Comment by eye-dee** [comment]
Hello @mirzemehdi 

Thanks for checking my issue. I have investigated a bit what happens, but up until now I couldn't find any explanation.
```
This error is also same as this https://github.com/mirzemehdi/KMPAuth/issues/19. New Credential Manager doesn't support some devices, I am planning to implement old credential manager when this issue happened thanks to @stevdza-san 's suggestion.
```
I haven't seen mentioned errors on my side, not sure if that issue is related, but I will check again

```
Can you please let me know if this happens when you sign-in with google in device and trying again?
```
This happens always regardless of what I do
To give more details: 
1. I have tried it on my real device, where I have multiple accounts in credential manager and in other applications similar functionalities work(see the screenshot attached)
![image](https://github.com/mirzemehdi/KMPAuth/assets/12158855/7ac4f71e-0474-4664-9d77-d9ca0657e785)

2. When trying an emulator it gives the same error (No Credentials available). But there I can understand it since on fresh emulator there is no accounts signed in. However the same happens when I am signing in settings or photos app. The most strange thing for me is that when I shared account that I use in emulator to sign in with my friend, he did the same thing (run KMPAuth example on emulator with my account singed in settings) and for him it worked fine
